recipes/trackhub/0.1.2019.12.24/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "trackhub" %}
+{% set version = "0.1.2019.12.24" %}
+{% set commit = "157ac4d3c392bd8c47b675a68f3c1a25b08a4afa" %}
+{% set sha256 = "4693509121d7e3eb2d2750a59af6b3fdfd391cc90519a9931add77aa588f80e3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # "stale" PR here: https://github.com/daler/trackhub/pull/33
+  url: https://github.com/daler/trackhub/tarball/{{ commit }}
```

Comment:
The url should be `https://github.com/daler/trackhub/archive/{{ commit }}.tar.gz`